jsm-aug2023.qmd

```diff
@@ -0,0 +1,237 @@
+- SAS has 14 spatial covariance structures compared to the spatial exponential one available in mmrm.
+:::
+
+::: {.column width="40%"}
+::: callout-tip
+For users that need more structures, `mmrm` is easily extensible - let us know via [feature requests](https://github.com/openpharma/mmrm/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=) in the GitHub repository.
+:::
+:::
+:::
+
+## Covariance structures details
+
+|       Covariance structures                   | `{mmrm}`   |  `PROC MIXED`  |
+|:----------------------------------------------|:------------:|:-----:|
+|     Unstructured (Unweighted/Weighted)        |     X/X      |  X/X  |
+|     Toeplitz (hetero/homo)      |     X/X      |  X/X  |
+| Compound symmetry (hetero/homo) |     X/X      |  X/X  |
+|  Auto-regressive (hetero/homo)  |     X/X      |  X/X  |
+|  Ante-dependence (hetero/homo)  |     X/X      |  X    |
+|        Spatial exponential      |     X        |  X    |
+
+## Degrees of Freedom Methods
+
+|       Method     |  `{mmrm}`      | `PROC MIXED` |
+|:---------------- |:----------:|:---:|
+|       Contain         |     X*     |  X  |
+|       Between/Within  |     X*     |  X  |
+|       Residual        |     X*     |  X  |
+|       Satterthwaite   |     X      |  X  |
+|       Kenward-Roger   |     X      |  X  |
+|       Kenward-Roger (Linear)**   |  X|  X  |
+
+*Available through the `emmeans` package.
+
+**This is not identical to the KR(Lin) in `PROC MIXED`
+
+## Contrasts/LSMEANS
+
+Contrasts and least square means estimates are available in `mmrm` using:
+
+  - `mmrm::df_1d`, `mmrm::df_md` 
+  - S3 method that is compatible with `emmeans`
+  - LS means difference can be produced through `emmeans` (`pairs` method)
+  - Degrees of freedom method is passed from `mmrm` to `emmeans`
+  - By default `PROC MIXED` and `mmrm` do not adjust for multiplicity, whereas `emmeans` does.
+
+## Benchmarking with other R packages
+
+- Compared `mmrm::mmrm` with `nlme::gls`, `lme4::lmer`, `glmmTMB::glmmTMB`
+- Highlights 
+  - `mmrm` has faster convergence time
+  - `mmrm` and `gls` estimates have smaller differences from SAS `PROC MIXED` estimates
+  - `mmrm` and `gls` are more resilient to missingness
+- Detailed results at the online [comparison vignette](https://openpharma.github.io/mmrm/main/articles/mmrm_review_methods.html)
+
+## Impact of `mmrm`
+
+- CRAN downloads: 382 in the last week (7/31/2023 – 8/6/2023)
+  - [https://cran.r-project.org/web/packages/mmrm/](https://cran.r-project.org/web/packages/mmrm/) 
+
+- GitHub repository: 60 stars as of 8/7/2023
+  - [https://github.com/openpharma/mmrm](https://github.com/openpharma/mmrm) 
+
+
+## Outlook
+
+- We still have major features on our backlog
+  - Type II and type III ANOVA tests
+- Working on comparison to other implementations
+  - Parameter estimates, computation time, convergence, ...
+- Please let us know what is missing in `mmrm` for you! And try it out :-)

```

Comment:
you could add another bullet point for new CRAN version 0.3 coming soon and include few additional features from https://github.com/openpharma/mmrm/blob/main/NEWS.md#new-features